tests/gold_tests/pluginTest/slice/slice_error.test.py

```diff
@@ -294,7 +294,6 @@
 ts.Disk.records_config.update({
 #  'proxy.config.diags.debug.enabled': 1,
 #  'proxy.config.diags.debug.tags': 'slice',
-  'proxy.config.http.cache.http': 0,
   'proxy.config.http.wait_for_cache': 0,
```

Comment:
Should this line go away with the enable_cache setting?

Reply:
Good point.